packages/extension-browser/src/devtools/views/app.css

```
@@ -36,20 +36,16 @@
     font-size: 12px; /* Match default overrides in Chrome */
     margin: 0;
     overflow: auto;
+    padding-left: 2.5rem;
     position: absolute;
     top: 0;
     left: 0;
     right: 0;
     bottom: 0;
 }
 
-.root a[href] {
+a[href] {
```

These should remain scoped to avoid bleeding if we re-use this code in a shared context, though I'm guessing this is the cause of some of the specificity issues you ran into earlier. 
```suggestion
.root a[href] {
```